Scripts/fastlane/Fastfile

```diff
@@ -464,11 +464,11 @@ import "./ScreenshotFastfile"
     ios_merge_translators_strings(strings_folder:"../WordPress/Resources/")
     sh("cd ../.. && ./Scripts/extract-framework-translations.swift")
 
-    xcbuild(
-        workspace: "../WordPress.xcworkspace",
+    gym(
         scheme: "WordPress",
+        workspace: "../WordPress.xcworkspace",
         configuration: "Debug",
-        xcargs: "-sdk iphonesimulator SYMROOT='/var/tmp/'"
+        destination: "platform=iOS Simulator,name=iPhone 11",
```

Comment:
Hmm... why this? We need the x86 build in /var/tmp because we upload it to Appetize (see the next line in the script).

Reply:
Or, we have to update the next line if we build to a different folder.

Reply:
Heh sorry – left in the middle of this – I thought the python change was the only one required here. Need to move it to `gym` or else it fails the same way the other one did. I'll take a look at this again for review.

Sorry for the spam – I'll move it back to `draft` until it's good to go 😓